common/lib/xmodule/setup.py

```diff
@@ -65,8 +62,7 @@
     # See https://setuptools.readthedocs.io/en/latest/setuptools.html#dynamic-discovery-of-services-and-plugins
     # for a description of entry_points
     entry_points={
-        'xblock.v1': XMODULES + XBLOCKS,
```

Comment:
:fire:

Reply:
![Nuke it from orbit](http://www.ericaricardo.com/blog/140225-nuke-it.jpg)